### PR TITLE
Fixing Image upload job

### PR DESCRIPTION
**Pull request by Shilpi-Das1**
Job Link:https://jenkins.ppc64le-cloud.org/job/temp-folder-testing-jobs/job/jenkins-backup/16/console

### CODE REVIEW

File: jobs/pipelines/powervs/powervs-image-create-and-upload-job/Jenkinsfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ pipeline {
 	    string(defaultValue: '', description: 'Rhel Url', name: 'RhelUrl')
 	    string(defaultValue: '', description: 'Rhcos Url', name: 'RhcosUrl')
 	    string(defaultValue: '', description: 'Centos Url', name: 'CentosUrl')
-        string(defaultValue: 'v0.1.1-alpha.6', description: "Pvsadm version", name: 'pvsadmVersion')
+        string(defaultValue: 'v0.1.2-alpha.2', description: "Pvsadm version", name: 'pvsadmVersion')
```
Comment (sudeeshjohn) [inline jobs/pipelines/powervs/powervs-image-create-and-upload-job/Jenkinsfile:8]: can you please fix this indentation ?

Comment (Shilpi-Das1) [reply jobs/pipelines/powervs/powervs-image-create-and-upload-job/Jenkinsfile]: @sudeeshjohn Fixed it https://github.com/ppc64le-cloud/jenkins-infra/blob/95f88d45de50a02a45e19daf4857f043430d24dc/jobs/pipelines/powervs/powervs-image-create-and-upload-job/Jenkinsfile

**Review by sudeeshjohn — APPROVED** [review]
/lgtm

**Pull request merged**